src/mpid/ch4/include/mpidpre.h

```diff
@@ -219,6 +227,8 @@ typedef struct {
     struct MPIR_Request *anysource_partner_request;
 #endif
 
+    MPIDIG_gpu_req_t gpu;
+
```

Comment:
Why do we need it here? I thought buffer is not part of the persistent request.

Reply:
I think we had this discussion already, this is common to netmod and shmmod so should be accessible regardless the path used (either direct netmod or AM)